jobs/build/openshift-scripts/scripts/merge-and-build-openshift-scripts.sh

```diff
@@ -130,15 +130,10 @@ else
     echo "=========="
     echo "Build and Push libra repos"
     echo "=========="
-    if [ "${BUILD_MODE}" == "online:stg" ] ; then
-        ssh ocp-build@rcm-guest.app.eng.bos.redhat.com "/mnt/rcm-guest/puddles/RHAOS/scripts/libra-repo-to-mirrors.sh stg"
-    elif [ "${BUILD_MODE}" == "online:int" ] ; then
-        ssh ocp-build@rcm-guest.app.eng.bos.redhat.com "/mnt/rcm-guest/puddles/RHAOS/scripts/libra-repo-to-mirrors.sh int"
-    elif [ "${BUILD_MODE}" == "release" ] ; then
-        ssh ocp-build@rcm-guest.app.eng.bos.redhat.com "/mnt/rcm-guest/puddles/RHAOS/scripts/libra-repo-to-mirrors.sh release"
-    else
-        echo "Unknown BUILD_MODE: ${BUILD_MODE}"
-    fi
+    repo_mode="$(echo ${BUILD_MODE} | cut -d ":" -f 2)"
```

Comment:
We can just change libra-repo-to-mirrors.sh to accept $BUILD_MODE directly (i.e. "online:int" vs "int") as an argument.